crates/fuel-core/src/schema/tx/input.rs

```diff
@@ -196,7 +196,8 @@ impl From<&fuel_tx::Input> for Input {
                 tx_pointer: TxPointer(*tx_pointer),
                 witness_index: Default::default(),
                 predicate_gas_used: (*predicate_gas_used).into(),
-                predicate: HexString(predicate.clone()),
+                // TODO[RC]: Not yet sure why these are needed?
```

Comment:
Taking a guess here, but I think it's because converting to hex format ensures that all characters are in utf-8, which is an  assumption of most json decoders.

Reply:
My conjecture is that [this recent change](https://github.com/FuelLabs/fuel-vm/pull/670/files#diff-5a170feaca83c54fdc3cde01fd6b6bdfe23d0f7c983e5f1bf8d6fd5d18328353R84) caused this code to require `.to_vec()` now.